jujugui/static/gui/src/app/state/test-handlers.js

```diff
@@ -0,0 +1,126 @@
+/* Copyright (C) 2018 Canonical Ltd. */
+'use strict';
+
+const StateChangeHandlers = require('./handlers');
+const State = require('./state');
+
+const jujulibTestHelper = require('@canonical/jujulib/api/test-helpers');
+const AllWatcherResponse = require('@canonical/jujulib/tests/data/allwatcher-response');
+
+function setupApp(options = {}) {
+  return {
```

Comment:
The goal is to have as small of a surface area to interact with app over time but a lot of these are required simply for legacy interactions and will be removed.